_docs/architecture.md

```diff
@@ -78,6 +78,22 @@ The following are examples of some possible webhook flows and events. Please not
 
 You can obtain an up-to-date list of available webhook events using the [Webhook Events endpoint](https://api.transferzero.com/documentation/#/Webhooks/get-webhook-events) of our API.
 
+## Collections
+We have available several type of webhooks that you can subscribe to that are you may find useful when doing collections. These webhooks start with `payin_method.`
+
+Here are the webhoks available:
+* `payin_method.incomplete`: This webhook is sent out in case there is some missing details that we require you to sent in order to proceed
+* `payin_method.pending`: This webhook is sent out to notify you of a collection request initiated. During this time we are waiting for the Sender to send in the money
+* `payin_method.processing`: This webhook is sent out when we have received funds from the Sender and are still awaiting for a full confirmation from the provider
+* `payin_method.paid_in`: This webhook is sent out to notify that the collection from the Sender was successful
+* `payin_method.error`: This webhook is sent out when a collection attempt has failed
+* `payin_method.refunded`: This webhook is sent out to notify that we successfully refunded the money to the Sender
+* `payin_method.canceled`: This webhook is sent out to notify that we successfully cancelled the Collection and are currently attempting to refund the Sender in case the money were already sent to us
```

Comment:
```suggestion
* `payin_method.canceled`: This webhook is sent out to notify that we successfully cancelled the Collection and are currently attempting to refund the Sender in case the money was already sent to us
```